airdrop_utils/contracts/NFTAirdrop.sol

```diff
@@ -0,0 +1,30 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.4;
+
+import "hardhat/console.sol";
+import "@spanning/contracts/utils/Counters.sol";
+import "@spanning/contracts/token/ERC721/extensions/ERC721Enumerable.sol";
+
+contract NFTAirdrop is SpanningERC721Enumerable {
+    using Counters for Counters.Counter;
+    
+    Counters.Counter private _tokenIds;
+    
+    constructor() ERC721("NFT Aidrop Demo", "NAD") {
```

Comment:
does this need to be SpanningERC721 in the delegated constructor?

Reply:
Done